src/platform-implementation-js/dom-driver/gmail/views/gmail-route-view/gmail-route-view.ts

```diff
@@ -387,6 +389,14 @@ class GmailRouteView {
         );
       }, 15_000);
     } catch {
+      if (isStreakAppId(this._driver.getAppId())) {
+        this._driver
+          .getLogger()
+          .error(new Error("Thread container for preview pane wasn't found"), {
+            html: censorHTMLstring(previewPaneContainer.innerHTML),
+          });
+      }
+
       throw new SelectorError(selector_2023_11_16, {
         cause: new Error("Thread container for preview pane wasn't found"),
       });
```

Comment:
The error will get logged twice for Streak. This would fix it:

```suggestion
      const selectorError = new SelectorError(selector_2023_11_16, {
        cause: new Error("Thread container for preview pane wasn't found"),
      });
      if (isStreakAppId(this._driver.getAppId())) {
        this._driver
          .getLogger()
          .error(selectorError, {
            html: censorHTMLstring(previewPaneContainer.innerHTML),
          });
      }

      throw selectorError;
```

Reply:
updated